### PR TITLE
ICDS Dashboard: Corrections in UI and reports

### PR DESCRIPTION
**Pull request by saiprakash-v**
https://app.asana.com/0/1183228534052225/1184201618678897
https://app.asana.com/0/1185205808600689/1184225472636119
https://app.asana.com/0/1185205808600689/1184225472636113
##### SUMMARY
**Out of school Adolescent girls:**
Screenshot from 2020-07-20 13-48-31

**AWC Infrastructure weighing scale:**
Screenshot from 2020-07-22 13-09-44

**Service Delivery Report changes:**
**Before:**
[Service Delivery Report__22-07-2020__07-34-50.xlsx](https://github.com/dimagi/commcare-hq/files/4958578/Service.Delivery.Report__22-07-2020__07-34-50.xlsx)
**After:**
[Service Delivery Report__22-07-2020__07-13-16.xlsx](https://github.com/dimagi/commcare-hq/files/4958585/Service.Delivery.Report__22-07-2020__07-13-16.xlsx)

### CODE REVIEW

**Comment by saiprakash-v** [comment]
@ShubhamBansal1997 Tagging to review

File: custom/icds_reports/utils/__init__.py [hunk 1]
```diff
@@ -1475,12 +1475,19 @@ def create_service_delivery_report(excel_data, data_type, config):
                                                       'Growth Monitoring (Children 3-5 years)',
                                                       ]
 
-    secondary_headers = ['Not provided',
-                         'Provided for 1-7 days',
-                         'Provided for 8-14 days',
-                         'Provided for 15-20 days',
-                         'Provided for 21-24 days',
-                         'Provided for at least 25 days (>=25 days)']
+    secondary_headers_sn = ['Not provided',
```
Comment (ShubhamBansal1997) [inline custom/icds_reports/utils/__init__.py:1478]: nit: p should be a capital letter.

**Review by ShubhamBansal1997 — COMMENTED** [review]
Overall looks good left a minor comment, good to test on staging first before moving to tcl

**Review by ShubhamBansal1997 — APPROVED** [review]

**Comment by saiprakash-v** [comment]
Tested by Zohaib on staging and changes looks good

**Pull request merged**